tests/phpunit/tests/functions/TestConfigWpHome.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Test the _config_wp_home function
+ *
+ * @group Functions
+ * @covers ::_config_wp_home
+ */
+class Test_for_config_wp_home extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 57179
+	 * @dataProvider _config_wp_home_dataset
```

Comment:
```suggestion
	 * @ticket 57179
	 *
	 * @dataProvider data_config_wp_home
	 *
	 * @param string $url      The URL to pass to _config_wp_home().
	 * @param string $expected The expected result.
```
- Data providers should have the `data_` prefix. See [this ticket](https://core.trac.wordpress.org/ticket/53119).
- Parameters should be documented in the test method docblock.